test/extended/dns/dns.go

```diff
@@ -294,6 +294,7 @@ var _ = Describe("DNS", func() {
 	f := e2e.NewDefaultFramework("dns")
 
 	It("should answer endpoint and wildcard queries for the cluster [Conformance]", func() {
+		Skip("TODO: re-enable when wildcard dns queries are supported by openshift dns operator")
```

Comment:
Instead of changing it here, add this to test/extended/util/test.go and add it to the section on broken tests with a link to the enablement. 

No one should be adding Skip rules to their tests.

Reply:
edit: unless they are conditional skips (this isn't conditional)